runtime/src/bank.rs

```diff
@@ -1723,6 +1723,7 @@ mod tests {
         system_instruction, system_program,
         sysvar::{fees::Fees, rewards::Rewards},
     };
+    use solana_stake_program::stake_state;
```

Comment:
prefer putting self inside curlies on the next line to a duplicate line

Reply:
@rob-solana Done.